src/effects/audiounits/AUControl.mm

```diff
@@ -291,10 +291,14 @@ - (void)cocoaViewResized:(NSNotification *)notification
             if (auto factoryClass = [bundle classNamed:viewClass])
                // Create an instance of the class
                if (id factoryInst = [[[factoryClass alloc] init] autorelease])
-                  // Create the view, suggesting a reasonable size
-                  if ((mView = [factoryInst uiViewForAudioUnit:mUnit
-                                                 withSize:NSSize{800, 600}]))
+                 // Create the view, suggesting a reasonable size
+                 try {
+                   if ((mView =
```

Comment:
I don't understand why this works. I assume, that you try to catch Obj-C exception, and the syntax is different https://stackoverflow.com/questions/4648952/objective-c-exceptions

Reply:
I guess because objective-c++ exceptions are derived from c++ exceptions.